src/Compilers/CSharp/Portable/CSharpExtensions.cs

```diff
@@ -116,6 +116,22 @@ public static bool Any<TNode>(this SeparatedSyntaxList<TNode> list, SyntaxKind k
             return list.IndexOf(kind) >= 0;
         }
 
+        internal static int Count<TNode>(this SeparatedSyntaxList<TNode> list, Func<TNode, bool> predicate)
```

Comment:
>internal static int Count<TNode>(this SeparatedSyntaxList<TNode> list, Func<TNode, bool> predicate) [](http://example.com/codeflow?start=8&length=99)

Is there something C# specific in this helper?

Reply:
No, and I'm very amenable to moving this wherever you think appropriate. I saw the linq-like extension methods right above this one, so that is where I put this, but if you think there's a better place for this, I'll go ahead and move it.

Reply:
Since there is nothing C# specific in this helper, I think it should not be added to CSharpExtensions. Please look for a better place so that VB code could also take advantage of the helper.

Reply:
Moved to a non-C# specific location, but kept as an extension method.